src/lib/api.js

```diff
@@ -0,0 +1,114 @@
+import axios from 'axios';
+import pickBy from 'lodash/pickBy';
```

Comment:
this is 4.5k minified and gzipped. Can we avoid it? https://bundlephobia.com/result?p=lodash.pickby@4.6.0